### DISCUSSION TITLE
kubectl添加的ingress路由

### DISCUSSION THREAD

**Discussion by auqf** [alibaba/higress#299] [Q&A]
在图形化控制台无法查看到kubectl添加的路由规则
![image](https://user-images.githubusercontent.com/8895540/232746189-90310069-2093-4784-8a34-359a36d29582.png)
![image](https://user-images.githubusercontent.com/8895540/232746273-c97d2b42-e74d-4633-9d21-55b7b495f8b0.png)

**Comment by CH3CHO** [comment]
控制台添加的路由上面是有特殊标记的。人工添加的路由是无法再控制台上编辑的。这也是为了避免控制台意外将人工编辑的路由中自己不支持的配置给覆盖掉。

**Reply by auqf** [reply]
如果两边写的类似的路由冲突了，哪边的优先级高呢？另外出现这种问题，在控制台无法进行诊断排障

**Reply by CH3CHO** [reply]
麻烦 @johnlanni @gengleilei 帮忙回答一下。

**Reply by CH3CHO** [reply]
找到了文档，你可以看一下：[路由规则的优先级](https://github.com/alibaba/higress/files/11307815/default.pdf)

**Reply by Gourds** [reply]
提个小建议：通过kubectl 创建的higress条目，最好可以和console同步，怕修改可以加个保护按钮或者禁止修改只能查看。这样的话两边信息就同步了，不然使用感受有些别扭

**Reply by johnlanni** [reply]
有计划做的，但是目前社区没有足够的前端贡献者，所以延后了
